ablySpec/RealtimeClientConnection.swift

```diff
@@ -1307,6 +1307,66 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15a
+                it("should attempt to reconnect immediately and try to restore the connection") {
```

Comment:
If this test is testing the failure condition, this description should reflect that.